src/queue.ts

```diff
@@ -177,6 +177,7 @@ export class Queue<T = never> {
     object: Record<M, () => void>,
     method: M
   ): void {
+    /* eslint-disable-next-line @typescript-eslint/no-this-alias */
```

Comment:
Maybe we should disable `@typescript-eslint/no-this-alias` !?

Reply:
You should remove the `me` variable and use an arrow function here. That would be cleaner and solve the problem:
```js
    const original = object[method];
    if (!original) {
      throw new Error("Method " + method + " undefined");
    }

    object[method] = (...args: unknown[]): void => {
    // add this call to the queue
    this.queue({
        args: args,
        fn: original,
        context: this
      });
    };
  }
```

Reply:
There is this being used in the function. Arrow functions don't have this therefore we would lose the context (likely breaking change).

Aliasing this is generally a bad idea but in some edge cases (like this one) it's the simplest solution.

Reply:
Ah. I overlook the "context". All good!